xml/art_sle_ha_pmremote.xml

```diff
@@ -0,0 +1,428 @@
+        cluster nodes.</para>
+      </step>
+      <step>
+       <para>Create an authentication key to all cluster and remote nodes:</para>
+       <screen>&prompt.root;<command>mkdir</command> -p --mode=0755 /etc/pacemaker
+&prompt.root;<command>dd</command> if=/dev/urandom of=/etc/pacemaker/authkey bs=4k count=1</screen>
+      </step>
+      <step>
+       <para>If you want to enable the <systemitem class="daemon"
+        >pacemaker_remote</systemitem> service, run:</para>
+       <screen>&prompt.root;<command>chkconfig</command> pacemaker_remote on</screen>
+      </step>
+      <step>
+       <para>Restart &pace; with <command>systemctl restart pacemaker</command>.</para>
+      </step>
+      <step>
+       <para>Insert the following line in the <filename>/etc/csync2/csync2.cfg</filename>:</para>
+       <screen>include /etc/pacemaker/authkey</screen>
+      </step>
+      <!-- Run csync2? -->
+     </substeps>
+    </step>
+    <!--<step>
+    <para>Check if all necessary firewall ports for an HA cluster are open
+     (for example, 3121, and more)</para>
+   </step>-->
+    <step>
+     <para>Define a primitive resource per remote node:</para>
+     <screen>&prompt.root;<command>crm</command> configure primitive suse09</screen>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="sec.ha.pmremote.config.kvm.guest">
+   <title>Configuring the KVM Guests</title>
+   <para>
+    The following procedure creates a KVM guest node <systemitem
+      class="domainname">&wsII;</systemitem> on your physical host
+    <systemitem class="domainname">&wsI;</systemitem>:
+   </para>
+   <procedure>
+    <step>
+     <para>Create a KVM guest on <systemitem class="domainname"
+      >&wsI;</systemitem>. Make sure to configure the guest with
+     a hostname and a static IP address. </para>
+    </step>
+    <step>
+     <para>Add the KVM guest hostname <systemitem
+      class="domainname">&wsII;</systemitem> and its IP address to the hosts
+     machine at <filename>/etc/hosts</filename>.</para>
+    </step>
+    <step>
+     <para>Configure Firewall settings on your guest node.</para>
+    </step>
+    <step>
+     <para>Verify connectivity between guests and host by using
+      <command>ping</command> and <command>ssh</command>.</para>
+    </step>
+    <step>
+     <para>Install the <package>pacemaker-remote</package> package and
+     enable it:</para>
+     <screen>&prompt.root;<command>zypper</command> in pacemaker-remote
+&prompt.root;<command>systemctl</command> enable pacemaker_remote</screen>
+    </step>
+    <step>
+     <para>Copy the authentication key from your physical host to your guest
+      node:</para>
+     <screen>&prompt.root;<command>mkdir</command> -p --mode=0750 /etc/pacemaker
+&prompt.root;<command>chgrp</command> haclient /etc/pacemaker
+&prompt.root;<command>scp</command> root@&wsI;:/etc/pacemaker/authkey /etc/pacemaker</screen>
+    </step>
```

Comment:
Oh yes! I splitted the above step into two:

6. Create the /etc/pacemaker directory and change the group:

        root # mkdir -p --mode=0750 /etc/pacemaker
        root # chgrp haclient /etc/pacemaker


7. Run `csync2` to copy the authentication key from your physical host to your guest node:

        root # csync -xv